sentences/sv/homeassistant_HassTurnOff.yaml

```diff
@@ -3,6 +3,20 @@ intents:
   HassTurnOff:
     data:
       - sentences:
-          - <slå_av> <name>
-          - <stäng_gardiner> <name>
+          - <slå_av> <name> # turn off
+        excludes_context:
+          domain:
+            - light
+            - cover
+      - sentences:
+          - <stäng_gardiner> <name> #close curtains
+        requires_context:
+          domain:
+            - cover
         response: cover
+      - sentences:
+          - <slå_av> <name> # lights off
```

Comment:
I think that one issue can be that the expansion rule for `<slå_av>` is a bit to wide, in contains both the work turn off, stopp, shutoff etc in swedish and for lights it might make more since to split it up in multiple sentences.
like: 
"släck <name>"
looking at the DA language.